### PR TITLE
Feature/CVSB-15169 - [BUGS] found on CVSB-10283

### PR DESCRIPTION
**Pull request by irina005**
Bug tickets:

https://jira.dvsacloud.uk/browse/CVSB-15169
https://jira.dvsacloud.uk/browse/CVSB-15184

**Fixed:**

- UTC format to correctly display backend date values 
-  "advisory" defect background to be white with black writing
- "advisory" and "dangerous" defects types to be listed on the same line with the name of the defect
- capitalize just the first letter in a string containing multiple words
- check html ids do not contain white spaces
- bug on display emission details section when test result is not pass

### CODE REVIEW

File: src/app/technical-record/adr/adr-details/adr-details.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h2 class="govuk-heading-m">ADR details</h2>
       <dt class="govuk-summary-list__key">Date processed</dt>
       <dd id="test-ADR-approvalDate" class="govuk-summary-list__value">
         <span *ngIf="adrDetails.vehicleDetails?.approvalDate; else hyphen">
-          {{ adrDetails.vehicleDetails?.approvalDate | date: 'dd/MM/yyyy' }}
+          {{ adrDetails.vehicleDetails?.approvalDate | date: 'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/adr/adr-details/adr-details.component.html:17]: not every date field need UTC. Please check the API and only use on date fields with time stamp

File: src/app/technical-record/adr/adr-details/__snapshots__/adr-details.component.spec.ts.snap [hunk 2]
```diff
@@ -44,7 +44,7 @@ exports[`AdrDetailsComponent should create view only with populated data 1`] = `
       >
         
         <span>
-           10/03/2020 
+           09/03/2020
```
Comment (alleypa) [inline src/app/technical-record/adr/adr-details/__snapshots__/adr-details.component.spec.ts.snap:47]: why changing this

File: src/app/technical-record/adr/tank-inpections/__snapshots__/tank-inpections.component.spec.ts.snap [hunk 3]
```diff
@@ -76,7 +76,7 @@ exports[`TankInpectionsComponent should create view only with populated data 1`]
               >
                 
                 <span>
-                   18/03/2020 
+                   17/03/2020
```
Comment (alleypa) [inline src/app/technical-record/adr/tank-inpections/__snapshots__/tank-inpections.component.spec.ts.snap:79]: why the change

File: src/app/technical-record/adr/tank-inpections/tank-inpections.component.html [hunk 4]
```diff
@@ -23,7 +23,7 @@ <h2 class="govuk-heading-m">Tank inspections</h2>
             </td>
             <td id="test-tc2IntermediateExpiryDate" class="govuk-table__cell">
               <span *ngIf="tankDetails?.tc2Details?.tc2IntermediateExpiryDate; else hyphen">
-                {{ tankDetails?.tc2Details?.tc2IntermediateExpiryDate | date:'dd/MM/yyyy' }}
+                {{ tankDetails?.tc2Details?.tc2IntermediateExpiryDate | date:'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/adr/tank-inpections/tank-inpections.component.html:26]: not needed to change

File: src/app/technical-record/adr/tank-inpections/tank-inpections.component.html [hunk 5]
```diff
@@ -40,7 +40,7 @@ <h2 class="govuk-heading-m">Tank inspections</h2>
             </td>
             <td id="test-tc3PeriodicExpiryDate-{{ i }}" class="govuk-table__cell">
               <span *ngIf="tc3Details.tc3PeriodicExpiryDate; else hyphen">
-                {{ tc3Details.tc3PeriodicExpiryDate | date:'dd/MM/yyyy' }}
+                {{ tc3Details.tc3PeriodicExpiryDate | date:'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/adr/tank-inpections/tank-inpections.component.html:43]: same

File: src/app/technical-record/authorisation-into-service/authorisation-into-service.component.html [hunk 6]
```diff
@@ -3,7 +3,7 @@
     <dt class="govuk-summary-list__key">COC issue date</dt>
     <dd id="test-authIntoService-cocIssueDate" class="govuk-summary-list__value">
       <span *ngIf="authoIntoService?.cocIssueDate; else hyphen">
-        {{ authoIntoService?.cocIssueDate | date: 'dd/MM/yyyy' }}
+        {{ authoIntoService?.cocIssueDate | date: 'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/authorisation-into-service/authorisation-into-service.component.html:6]: please revert `"authIntoService": {
          "cocIssueDate": "2020-05-15",
          "dateReceived": "2020-05-15",
          "datePending": "2020-05-15",
          "dateAuthorised": "2020-05-15",
          "dateRejected": "2020-05-15"
        }`

 As per the API

File: src/app/technical-record/authorisation-into-service/authorisation-into-service.component.html [hunk 7]
```diff
@@ -42,7 +42,7 @@
     <dt class="govuk-summary-list__key">Date rejected</dt>
     <dd id="test-authIntoService-dateRejected" class="govuk-summary-list__value">
       <span *ngIf="authoIntoService?.dateRejected; else hyphen">
-        {{ authoIntoService?.dateRejected | date: 'dd/MM/yyyy' }}
+        {{ authoIntoService?.dateRejected | date: 'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/authorisation-into-service/authorisation-into-service.component.html:45]: same for all this UTC change

File: src/app/technical-record/tech-rec-history/tech-rec-history.component.html [hunk 8]
```diff
@@ -30,16 +30,16 @@
       </td>
       <td id="{{ 'test-createdAt-' + i }}" class="govuk-table__cell">
         <span *ngIf="techRec.createdAt; else hyphen">
-          {{ techRec.createdAt | date: 'dd/MM/yyyy' }}
+          {{ techRec.createdAt | date: 'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/technical-record/tech-rec-history/tech-rec-history.component.html:33]: perfect. This is where you need UTC

File: src/app/technical-record/tech-rec-history/tech-rec-history.component.html [hunk 9]
```diff
@@ -30,16 +30,16 @@
...
         </span>
       </td>
       <td id="{{ 'test-lastUpdatedByName-' + i }}" class="govuk-table__cell">
         {{ techRec.lastUpdatedByName | DefaultNullOrEmpty | titlecase }}
       </td>
       <td id="{{ 'test-lastUpdatedAt-' + i }}" class="govuk-table__cell">
         <span *ngIf="techRec.lastUpdatedAt; else hyphen">
-        {{ techRec.lastUpdatedAt | date: 'dd/MM/yyyy'  }}
-        </span> 
+        {{ techRec.lastUpdatedAt | date: 'dd/MM/yyyy':'UTC'  }}
```
Comment (alleypa) [inline src/app/technical-record/tech-rec-history/tech-rec-history.component.html:41]: yes as well. Only for dates with timeStamp

File: src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html [hunk 10]
```diff
@@ -5,7 +5,7 @@
     </dt>
     <dd class="govuk-summary-list__value" id="test-seatbeltInstallationCheckDate">
       <span *ngIf="testType.seatbeltInstallationCheckDate; else hyphen">
-        {{ testType.seatbeltInstallationCheckDate | date: 'dd/MM/yyyy' }}
+        {{ testType.seatbeltInstallationCheckDate | date: 'dd/MM/yyyy':'UTC' }}
```
Comment (alleypa) [inline src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html:8]: please confirm this on test record. if no time stamp no need for UTC

File: src/app/technical-record-search/technical-record-search.component.html [hunk 11]
```diff
@@ -16,36 +16,19 @@ <h1 class="govuk-heading-xl">Search for a technical record</h1>
...
-              </div>
-
-              <div
-                *ngIf="
-                  searchError.includes('The provided partial VIN returned more than one match.')
-                "
-              >
-                Multiple vehicles found, search using the full vehicle identification number
-              </div>
+              <p class="white-space--normal">{{ searchError }}</p>
```
Comment (alleypa) [inline src/app/technical-record-search/technical-record-search.component.html:19]: who do we need white space. What are you trying to achieve. Do you need this `&nbsp;`

Comment (irina005) [reply src/app/technical-record-search/technical-record-search.component.html:19]: image

File: src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html [hunk 12]
```diff
@@ -4,9 +4,7 @@
       Carried out during test
     </dt>
     <dd class="govuk-summary-list__value" id="test-seatbeltInstallationCheckDate">
-      <span *ngIf="testType.seatbeltInstallationCheckDate; else hyphen">
-        {{ testType.seatbeltInstallationCheckDate | date: 'dd/MM/yyyy' }}
-      </span>
+        {{ testType.seatbeltInstallationCheckDate | DefaultNullOrEmpty }}
```
Comment (alleypa) [inline src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html:7]: should we not format this date field?

Comment (irina005) [reply src/app/test-record/seatbelt-installation-check/seatbelt-installation-check.component.html:7]: image

File: src/global.scss [hunk 13]
```diff
@@ -55,10 +55,23 @@
...
+
 .border--bottom-none {
   border-bottom: none;
 }
 
+.white-space--normal {
+  white-space: normal;
+}
+
+.link--blue:visited {
```
Comment (alleypa) [inline src/global.scss:71]: not needed I think

File: src/app/technical-record/test-history/test-history.component.html [hunk 14]
```diff
@@ -55,7 +55,7 @@
         </td>
         <td id="{{ 'test-view-' + i }}" class="govuk-table__cell">
           <a
-            class="govuk-link"
+            class="govuk-link link--blue"
```
Comment (alleypa) [inline src/app/technical-record/test-history/test-history.component.html:58]: try this `<a
                    href="javascript:void(0)"
                    class="govuk-link"`
